### ISSUE TITLE
Epsagon plugin to use a layer

### ISSUE THREAD

**Issue by JoaquinFernandez** [epsagon/serverless-plugin-epsagon#121]
It would be nice to have this plugin but instead on just wrapping a hander, setting a layer to every lambda, thus removing the need to include the epsagon dependecy. I have built that plugin and would be happy to share the code if interested.

**Comment by ranrib** [comment]
Hi @JoaquinFernandez, thanks for the suggestion. At this point, I think it would be easier to just add the layer for yourself through the `.yml` (either at the service or the function level):

```
layers:
    - arn:aws:lambda:us-east-1:xxxxxxxxxxxxx:layer:xxxxx:mylayer1
```

To get the latest node/python layer you can use this -  https://layers.epsagon.com/production?region=<region>&name=epsagon-<language>-layer&max_items=1


as an example from this tutorial for SAM - https://docs.epsagon.com/docs/integrations/deployment/sam

**Comment by JoaquinFernandez** [comment]
Thanks @ranrib for the response. That's actually what the plugin I wrote does. After a lot of testing, I ended up doing this:
1. Add the layer retrieved via the layers' endpoint
2. Add EPSAGON_TOKEN and EPSAGON_APP_NAME and EPSAGON_HANDLER (original file handler) as env vars
3. Update the actual lambda handler to '/opt/nodejs/node_modules/epsagon.wrapper'

Everything works as expected and lambdas are traced. But I wanted to know if that is not recommended or an anti-pattern in any way (I saw that's what the auto-tracing does, and imitate it).

Thank you and thanks for the prompt response
